port/common/omrport.tdf

```diff
@@ -1604,3 +1604,15 @@ TraceEvent=Trc_PRT_double_map_regions_Release_Restore_Region Overhead=1 Level=1
 TraceExit=Trc_PRT_double_map_regions_Release_Exit Group=double_map Overhead=1 Level=5 NoEnv Template="omrvmem_release_double_mapped_region returnCode: %d"
 TraceException=Trc_PRT_double_map_regions_Release_Failure Overhead=1 Level=1 Group=double_map NoEnv Template="Failed to mmap FIXED contiguous region of memory when releasing region"
 TraceException=Trc_PRT_double_map_regions_Release_Failure2 Overhead=1 Level=1 Group=double_map NoEnv Template="Failed to mmap FIXED contiguous region of memory. Expected address: %p, mmap returned: %p"
+
+TraceException=Trc_PRT_populateCgroupEntryListV1_fopen_failed Group=sysinfo Overhead=1 Level=1 NoEnv Template="populateCgroupEntryListV1: fopen failed for %s with errno=%d"
+TraceException=Trc_PRT_populateCgroupEntryListV1_unexpected_format Group=sysinfo Overhead=1 Level=1 NoEnv Template="populateCgroupEntryListV1: unexpected format of file %s"
+TraceEvent=Trc_PRT_populateCgroupEntryListV1_available_subsystems Group=sysinfo Overhead=1 Level=3 NoEnv Template="populateCgroupEntryListV1: available subsystems are 0x%llx"
+TraceException=Trc_PRT_populateCgroupEntryListV2_fopen_failed Group=sysinfo Overhead=1 Level=1 NoEnv Template="populateCgroupEntryListV2: fopen failed for %s with errno=%d"
+TraceException=Trc_PRT_populateCgroupEntryListV2_unexpected_format Group=sysinfo Overhead=1 Level=1 NoEnv Template="populateCgroupEntryListV2: unexpected format of file %s"
+TraceEvent=Trc_PRT_populateCgroupEntryListV2_available_subsystems Group=sysinfo Overhead=1 Level=3 NoEnv Template="populateCgroupEntryListV2: available subsystems are 0x%llx"
+TraceException=Trc_PRT_readCgroupSubsystemFile_unsupported_cgroup_version Overhead=1 Level=1 NoEnv Template="readCgroupSubsystemFile: cgroup or its version is unsupported"
+TraceException=Trc_PRT_sysinfo_cgroup_get_memlimit_memory_limit_scan_failed Group=sysinfo Overhead=1 Level=1 NoEnv Template="omrsysinfo_cgroup_get_memlimit: failed to scan memory limit from %s with portable error code=%d"
+TraceException=Trc_PRT_sysinfo_cgroup_get_memlimit_unsupported_cgroup_version Group=sysinfo Overhead=1 Level=1 NoEnv Template="omrsysinfo_cgroup_get_memlimit: cgroup or its version is unsupported"
+TraceException=Trc_PRT_populateCgroupEntryListV1_fgets_failed Group=sysinfo Overhead=1 Level=3 NoEnv Template="populateCgroupEntryListV1: fgets failed for %s with errno=%d"
+TraceException=Trc_PRT_populateCgroupEntryListV2_fgets_failed Group=sysinfo Overhead=1 Level=3 NoEnv Template="populateCgroupEntryListV2: fgets failed for %s with errno=%d"
```

Comment:
Mostly the difference is in the message `V1` or `V2`. If `V1`/`V2` are passed as an input parameter (%s), then the trace points can be reused between cgroup v1 and v2 support. It will be more sustainable if there are more iterations in the future.

Reply:
That occurred to me as well. If we do that, please don't use a string as the argument, use an integer instead.